### ISSUE TITLE
Gender-neutral language (wildcards) causing screen reader issues

### ISSUE THREAD

**Issue by PeterSchmidt23** [apache/superset#23964]
## Screenshot

![image](https://user-images.githubusercontent.com/130473712/236782444-a61c73de-3563-4bec-b449-16eec292635e.png)

## Description

It looks like some terms in the German language are translated gender neutral (with wildcard for the non-binary ones).
As a result, people with visual impairments can't use the frontend properly because the screen readers don't support it.
It is also not recommended by DBSV.
Attached is the information from the DBSV. 
https://www.dbsv.org/gendern.html

See ein\*e Benutzer\*in
## Design input
For better readability, the masculine term should be used.

**Comment by rusackas** [comment]
Well... actually... I'll re-open this. It seems legit, and maybe someone can tackle it. It's just stale though, so we might close it if nobody pitches in after a while.

**Comment by rusackas** [comment]
Not sure if this is on @eschutho or @yousoph 's accessibility project radar, but I will say this is at risk of being closed as stale if nobody wants to tackle it. Contributions warmly welcomed :D

**Comment by rusackas** [comment]
Closing as stale/not-planned. Contributions are still quite welcomed here if anyone becomes interested.
